apps/dexdrip/dexdrip.c

```diff
@@ -188,7 +187,7 @@ void print_packet(Dexcom_packet* pPkt) {
 
 void makeAllOutputs() {
     int XDATA i;
-    for (i=0; i < 16; i++) {
+    for (i=6; i < 16; i++) { // in the future, this should be set to only the channels being used for output, and add the one for input
```

Comment:
I did 0-5 so that when people are building theirs, they have the option to put it on any of the ADC pins they want.